overlord/snapstate/handlers.go

```diff
@@ -1169,15 +1204,31 @@ func (m *SnapManager) undoCopySnapData(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	// we migrated the data in this run - undo that
+	if snapsup.MigratedHidden {
+		if err := m.backend.UndoHideSnapData(snapsup.InstanceName()); err != nil {
```

Comment:
it's a bit asymmetric that the forward code has a helper and the backward code doesn't have a helper

Reply:
marking this are solved, the helper is gone